docker/build-docker-image.sh

```diff
@@ -1,9 +1,58 @@
 #!/bin/sh
-rm -f Dockerfile
-DIR_DOCKER_FILE=docker-centos-$1
-rm -rf $DIR_DOCKER_FILE
-mkdir  $DIR_DOCKER_FILE
-cp Dockerfile.template  $DIR_DOCKER_FILE/Dockerfile
-sed -i 's/{centosfrom}/'$1'/g' $DIR_DOCKER_FILE/Dockerfile
-sudo docker build $2 -t="erlang-rpm-build-"$1 $DIR_DOCKER_FILE 
-
+#
+#	
+#
+_docker_file="Dockerfile"
+_docker_temp="Dockerfile.template"
+_parameter1="$1"
+_parameter2="$2"
+_docker_dir="docker-centos-$_parameter1"
+#
+#
+#
+if [ -z $_parameter1 ]
+then
+	echo "
+Ops: parameters error
+first: version, ex: 6 or 7
+second: --no-cache [optional]
+-----------------------------------------
+Ex: ./build-docker-image.sh 7 --no-cache
+Ex: ./build-docker-image.sh 7
+Ex: ./build-docker-image.sh 6 --no-cache
+EX: ./build-docker-image.sh 6
+"
+	exit 1
+fi
+#
+#
+#
+if [ -e "$_docker_file" ]
+then
+	rm -f "$_docker_file"
+fi
+#
+#
+#
+if [ -e "$_docker_dir" ]
+then
+	rm -rf "$_docker_dir"
+	mkdir "$_docker_dir"
```

Comment:
this `mkdir "$_docker_dir"` should be outside from the `if`, otherwise if the directory does not exist the script does not create it. 

Something like:  

```
if [ -e "$_docker_dir" ]
then
  rm -rf "$_docker_dir"
fi

mkdir "$_docker_dir"
```